docs/cli/operator.mdx

```diff
@@ -846,8 +846,8 @@ Use the following options to change the behavior of this command.
 
 Update an existing Nexus Endpoint on the Server.
 
-An endpoint name is used in Workflow code to invoke Nexus operations.
-The endpoint target may either be a Worker, in which case `--target-namespace` and
+A Nexus Endpoint name is used in Workflow code to invoke Nexus Operations.  The
```

Comment:
Every sentence on one line - it helps when making comments and consistency across the docs repo. 

Other than that, looks good to merge.